rosetta-tools/pom.xml

```diff
@@ -72,6 +72,11 @@
 			<version>${project.version}</version>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>com.regnosys.rosetta</groupId>
+			<artifactId>rosetta-maven-plugin</artifactId>
+			<version>0.0.0.main-SNAPSHOT</version>
+		</dependency>
```

Comment:
Given my other comment, you can also remove this dependency and inject the common adaptor instead.